src/include/OSL/llvm_util.h

```diff
@@ -858,6 +861,10 @@ class OSLEXECPUBLIC LLVM_Util {
     llvm::Value *op_int_to_bool (llvm::Value *a);
     llvm::Value *op_float_to_double (llvm::Value *a);
     llvm::Value *op_int_to_longlong (llvm::Value *a);
+    // int_to_bool_to_int is essentially turning any nonzero -> 1
+    llvm::Value *op_int_to_bool_to_int (llvm::Value *a) {
+        return op_bool_to_int(op_int_to_bool(a));
```

Comment:
Does op_int_to_bool_to_int generate better code than opt_bool_to_int(op_ne(a, constant(0)))?
If so might be useful elsewhere.